lib/ui/src/components/sidebar/useLastViewed.ts

```diff
@@ -0,0 +1,51 @@
+import { window } from 'global';
+import { useCallback, useEffect, useState } from 'react';
+
+import { Selection, StoryRef } from './types';
+
+const retrieveLastViewedStoryIds = (): StoryRef[] => {
+  try {
+    const raw = window.localStorage.getItem('lastViewedStoryIds');
```

Comment:
FYI, we use https://www.npmjs.com/package/store2 in other parts of the code. not sure whether it matters.

Reply:
👍